frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetInstallSubmariner/InstallSubmarinerForm.tsx

```diff
@@ -362,8 +377,31 @@ export function InstallSubmarinerForm(props: { availableClusters: Cluster[] }) {
             })
         }
 
-        // if globalnet support create
-        if (!isGlobalnetAlreadyConfigured || !selectedClusters || !selectedClusters.length) {
+        if (anyUnsupported) {
+            resources.push({
+                apiVersion: ManagedClusterAddOnApiVersion,
+                kind: ManagedClusterAddOnKind,
+                metadata: {
+                    name: 'submariner',
+                    namespace: 'roks-cluster',
+                },
+                spec: {
+                    installNamespace: 'submariner-operator',
+                },
+            })
+            const broker: Broker = {
+                apiVersion: BrokerApiVersion,
+                kind: BrokerKind,
+                metadata: {
+                    name: 'submariner-broker',
+                    namespace: 'submariner-broker',
+                },
+                spec: {
+                    globalnetEnabled: false,
+                },
+            }
```

Comment:
This is also a hard-coded entry that probably shouldn't be in the code. `submariner-broker` would be the broker namespace for a ClusterSet named `submariner`.